dist/aframe-input-mapping-component.js

```diff
@@ -73,6 +74,13 @@
 
 	    // Controllers
 	    this.sceneEl.addEventListener('controllerconnected', function (event) {
+	      var matchedController = self.findMatchingController(event.detail.target);
+
+	      if (matchedController) {
```

Comment:
Super minor but id invert the control flow here so that updateControllerListeners only appears once.
```
let controllerObj = self.findMatchingController(event.detail.target);
if(!controllerObj ) {
  controllerObj = {...};
  self.loadedControllers.push(controllerObj );
}
self.updateControllersListeners(controllerObj );
```

Reply:
Done!